src/Report/EnabledMembers.php

```diff
@@ -0,0 +1,206 @@
+<?php declare(strict_types=1);
+
+namespace SilverStripe\MFA\Report;
+
+use SilverStripe\Forms\CheckboxField;
+use SilverStripe\Forms\DropdownField;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\NumericField;
+use SilverStripe\Forms\TextField;
+use SilverStripe\MFA\Extension\MemberExtension;
+use SilverStripe\MFA\Service\MethodRegistry;
+use SilverStripe\ORM\ArrayList;
+use SilverStripe\ORM\Filterable;
+use SilverStripe\ORM\SS_List;
+use SilverStripe\Reports\Report;
+use SilverStripe\Security\Member;
+use SilverStripe\View\ArrayData;
+
+if (!class_exists(Report::class)) {
+    return;
+}
+
+class EnabledMembers extends Report
+{
+    protected $title = 'Multi factor authentication status of members';
+
+    protected $description = 'Identifies all members on this site, with columns added to indicate the status each has'
+        . ' in regards to multi factor authentication method registration. E.g. how many methods are set up by that'
```

Comment:
```suggestion
        . ' in regards to multi-factor authentication method registration. E.g. how many methods are set up by that'
```

I'm not sure that the example provides enough value to warrant its important position on the screen TBH. When I look at it I get a little overwhelmed by what's going on at the top, and you get this example immediately by looking at the GridField anyway.